src/app/core/params.ts

```diff
@@ -74,7 +78,11 @@ export default class Params {
      * @memberof Params
      */
     isHeritable(key: string): boolean {
-        return this.heritable.includes(key);
+        if (typeof this.heritable[key] == "undefined") {
+            return false;
+        } else {
+            return true;
+        }
```

Comment:
Same thing, the idea is to simplify the changes and make the corresponding changes in device.ts